src/components/AddBook.js

```diff
@@ -15,47 +16,57 @@ const AddBook = () => {
   };
   const submitHandler = (e) => {
     e.preventDefault();
+
     const newBook = {
       item_id: uuidv4(),
       title: inputText.title,
       author: inputText.author,
-      category: 'default',
+      category: infoCategories[Math.floor(Math.random() * 100) % 4],
     };
 
     dispatch(asyncAddNewBook(newBook));
 
     setInputText({
       title: '',
       author: '',
+      category: '',
     });
   };
 
   return (
-    <>
-      <h3 className="heading">ADD NEW BOOK</h3>
+    <div className="addBookContainer">
+      <div className="dividerLow" />
+      <h3 className="form-title">ADD NEW BOOK</h3>
       <form className="form-container" onSubmit={submitHandler}>
-        <input
-          type="text"
-          className="inputTextBox"
-          placeholder="Book"
-          value={inputText.title}
-          name="title"
-          onChange={onChange}
-          required
-        />
-        <input
-          type="text"
-          className="inputTextBox"
-          placeholder="Author"
-          value={inputText.author}
-          name="author"
-          onChange={onChange}
-          required
-        />
-        <button type="submit"> Add Book</button>
+        <input className="inputTextBox input" placeholder="Book title" name="title" onChange={onChange} required value={inputText.title} />
+        <input className="inputTextBox input" placeholder="Author" name="author" onChange={onChange} required value={inputText.author} />
+        <button className="form-button" type="submit">ADD BOOK</button>
       </form>
-    </>
+    </div>
   );
 };
 
 export default AddBook;
+
+/* <h3 className="heading">ADD NEW BOOK</h3>
+<form className="form-container" onSubmit={submitHandler}>
+<input
+  type="text"
+  className="inputTextBox"
+  placeholder="Book"
+  value={inputText.title}
+  name="title"
+  onChange={onChange}
+  required
+/>
+<input
+  type="text"
+  className="inputTextBox"
+  placeholder="Author"
+  value={inputText.author}
+  name="author"
+  onChange={onChange}
+  required
+/>
+<button type="submit"> Add Book</button>
+</form> */
```

Comment:
- [ ] [OPTIONAL] Let's remove all the commented code to make our codebase look clean.